src/main/java/com/khadri/jdbc/prepare/data/movie/dao/MovieInsertDao.java

```diff
@@ -0,0 +1,36 @@
+package com.khadri.jdbc.prepare.data.movie.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Scanner;
+
+import com.khadri.jdbc.prepare.data.movie.Movie;
+
+public class MovieInsertDao {
+	public static void insertData(Movie mo) throws Exception {
+		// write JDBC logic here
+		Scanner sc = new Scanner(System.in);
+		Connection con = null;
+		PreparedStatement pstmt = null;
+
+		try {
+			Class.forName("com.mysql.cj.jdbc.Driver");
```

Comment:
Use the util class for connection refer others PR.

Reply:
used util class